lib/cli.js

```diff
@@ -119,7 +119,12 @@ exports.run = () => {
 	}
 
 	if (cli.flags.concurrency === '') {
-		throw new Error(colors.error(figures.cross) + ' The --concurrency and -c flags must be provided the maximum number of test files to run at once.');
+		throw new Error(colors.error(figures.cross) + ' The --concurrency and -c flags must be provided.');
+	}
+
+	if (cli.flags.concurrency &&
+		(!Number.isInteger(Number.parseFloat(cli.flags.concurrency)) || parseInt(cli.flags.concurrency, 10) < 0)) {
+		throw new Error(colors.error(figures.cross) + ' The --concurrency and -c flags must be a positive integer.');
```

Comment:
This should be `nonnegative integer`, since `0` is allowed. I feel a bit bad about this nitpick ☹️